scripts/install_packages

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package=""
+pip_package=""
+
+while getopts "c:p:q:" OPTCHAR ; do
+	case $OPTCHAR in
+		c )
+			action=$OPTARG
+			;;
+
+		p )
+			pip_package=$OPTARG
+			;;
+		
+		q )	
+			package=$OPTARG
+			;;
+	esac
+done
+
+if [ $DISTRO = "bionic" ]; then
+        if [ $action = "install" ]; then
+                apt-get update
+                apt-get install -y -q $package
+		pip3 install $pip_package
+                apt-get clean
+        elif [ $action = "uninstall" ]; then
+                apt-get autoremove --purge -y -q $package
+		apt-get clean
+                rm -rf /var/lib/apt/lists/*
+        fi
+
+elif [ $DISTRO = "centos" ]; then
+	if [ $action = "install" ]; then
+                yum update -y
```

Comment:
Let's use dnf instead of yum. dnf is more recent and default package manager used for centos8 and intended to be the replacement for YUM in RPM-based systems

Reply:
I would prefer to take this change as next level of optimization as part of seperate PR.

Reply:
Done